script.js

```diff
@@ -1,17 +1,434 @@
+     url_img: './snapshot-portfolio4.png',
+     text_content:
+       "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+     languages: ['html', 'css', 'javascript'],
+     button_live: {
+       url: '#',
+       icon_class: 'fa-life-ring',
+       btn_live: 'See live',
+     },
+     button_source: {
+       url: '#',
+       icon_class: 'fa-github-square',
+       btn_source: 'See Source',
+     },
+   },
+ };
+
+ // pop up section
+ const projectButton = document.querySelectorAll(
+   '#project1, #project2, #project3, #project4',
+ );
+ const headingTitle = document.querySelector('.pop-heading-det');
+ const titleInfo = document.querySelector('.title-info');
+ const contInfoDot = document.querySelector('.icon-dot-cont i');
+ const contInfoDot2 = document.querySelector('.icon-dot-cont2 i');
+ const jobInfo = document.querySelector('.job-info');
+ const yearInfo = document.querySelector('.year-info');
+ const imgPop = document.querySelector('.pop-img-proj');
+ const prjText = document.querySelector('.pop-text');
+ const langBtn = document.querySelectorAll('.lang-text');
+ const btnLiveText = document.querySelector('.live-btn-text');
+ const btnLiveIcon = document.querySelector('.live-btn-text + .fas');
+ const btnSourceText = document.querySelector('.source-btn-text');
+ const btnSourceIcon = document.querySelector('.source-btn-text + .fab');
+
+ // card section
+
+ projectButton.forEach((btn) => {
+   function popDetail() {
+     modal.style.display = 'block';
+     contPrjctDet.style.display = 'block';
+     headingTitle.textContent = projectInfo[btn.id].title;
+     titleInfo.textContent = projectInfo[btn.id].prj_info.title;
+     contInfoDot.classList.add(`${projectInfo[btn.id].prj_info.class}`);
+     jobInfo.textContent = projectInfo[btn.id].prj_info.job;
+     contInfoDot2.classList.add(`${projectInfo[btn.id].prj_info.class}`);
+     yearInfo.textContent = projectInfo[btn.id].prj_info.year;
+     imgPop.setAttribute('src', `${projectInfo[btn.id].url_img}`);
+     prjText.textContent = projectInfo[btn.id].text_content;
+     close.classList.add('fa-times');
+     for (let i = 0; i < projectInfo[btn.id].languages.length; i += 1) {
+       langBtn[i].textContent = projectInfo[btn.id].languages[i];
+     }
+     btnLiveText.textContent = projectInfo[btn.id].button_live.btn_live;
+     btnSourceText.textContent = projectInfo[btn.id].button_source.btn_source;
+     btnLiveIcon.classList.add(
+       `${projectInfo[btn.id].button_live.icon_class}`,
+     );
+     btnSourceIcon.classList.add(
+       `${projectInfo[btn.id].button_source.icon_class}`,
+     );
+   }
+   btn.addEventListener('click', popDetail);
+ });
+ function closeModal() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.